main.go

```diff
@@ -246,3 +288,32 @@ func parseTopicNamespace(topicNamespace string, databaseName string) string {
 
 	return s
 }
+
+func pqNotifyEventToString(ev pq.ListenerEventType) string {
+	switch ev {
+	case pq.ListenerEventConnected:
+		return "connected"
+	case pq.ListenerEventDisconnected:
+		return "disconnected"
+	case pq.ListenerEventConnectionAttemptFailed:
+		return "connection attempt failed"
+	case pq.ListenerEventReconnected:
+		return "reconnected"
+	default:
+		return "unknown"
+	}
+}
+
+func emptyNotificationsChannel(nc <-chan *pq.Notification, timeout time.Duration) {
```

Comment:
is this safe to call? I assume we discard all pending notifications and we will get the events anyway via the select statement in queue processor?